### DISCUSSION TITLE
upgrade tls1.0 to tls1.2 and we have issue in artisan queue:listen

### DISCUSSION THREAD

**Discussion by avirajguru9** [symfony/symfony#47154] [Q&A] [answered]
Hello All,

Recently we upgrade tls1.0 to tls1.2 due to this change we are facing mail isssue in crontab.

command run - php artisan queue:listen

**Error message -**

 [Symfony\Component\Process\Exception\RuntimeException]
  The process has been signaled with signal "11".
Exception trace:
 () at /users/mce00/web/html/vendor/symfony/process/Process.php:371
 Symfony\Component\Process\Process->wait() at /users/mce00/web/html/vendor/symfony/process/Process.php:203
 Symfony\Component\Process\Process->run() at /users/mce00/web/html/vendor/laravel/framework/src/Illuminate/Queue/Listener.php:120
 Illuminate\Queue\Listener->runProcess() at /users/mce00/web/html/vendor/laravel/framework/src/Illuminate/Queue/Listener.php:105
 Illuminate\Queue\Listener->listen() at /users/mce00/web/html/vendor/laravel/framework/src/Illuminate/Queue/Console/ListenCommand.php:73
 Illuminate\Queue\Console\ListenCommand->fire() at n/a:n/a
 call_user_func_array() at /users/mce00/web/html/bootstrap/cache/compiled.php:1257
 Illuminate\Container\Container->call() at /users/mce00/web/html/vendor/laravel/framework/src/Illuminate/Console/Command.php:169
 Illuminate\Console\Command->execute() at /users/mce00/web/html/vendor/symfony/console/Command/Command.php:256
 Symfony\Component\Console\Command\Command->run() at /users/mce00/web/html/vendor/laravel/framework/src/Illuminate/Console/Command.php:155
 Illuminate\Console\Command->run() at /users/mce00/web/html/vendor/symfony/console/Application.php:794
 Symfony\Component\Console\Application->doRunCommand() at /users/mce00/web/html/vendor/symfony/console/Application.php:186
 Symfony\Component\Console\Application->doRun() at /users/mce00/web/html/vendor/symfony/console/Application.php:117
 Symfony\Component\Console\Application->run() at /users/mce00/web/html/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php:107
 Illuminate\Foundation\Console\Kernel->handle() at /users/mce00/web/html/artisan:36

 

**Technical details**

PHP - 5.6
MySQL - 5.0
TLS - 1.2
Laravel - 5.2

**.env**
MAIL_DRIVER=mail
MAIL_HOST=smtp-auth.mpsa.com
MAIL_PORT=25
MAIL_ENCRYPTION=tls

Request you to all if anyone knows about this issue please help me.

Thank you !

**Comment by lyrixx** [comment, marked as answer]
Hello,

In the trace we can see:

> The process has been signaled with signal "11".

Which means

SIGSEGV | 11 | Invalid memory segment access (ANSI)
-|-|-

So the process you are running seems broken.

Unfortunately for you, this has nothing to do with Symfony, so we cannot help you more here.

Pour you can print `$process->getCommandLine()` to get the command line. Like that you'll be able to debug the process easily
